src/app/services/[service]/ServicePageClient.tsx

```
@@ -0,0 +1,57 @@
+  };
+
+  if (!service) {
+    notFound();
+  }
+
+  return (
+    <div>
+      <h1
+        id="title"
+        onBlur={handleContentChange}
+        dangerouslySetInnerHTML={{ __html: service.title || '' }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)